O365/connection.py

```diff
@@ -1103,7 +1095,7 @@ def naive_request(self, url: str, method: str, **kwargs) -> Response:
             # lazy creation of a naive session
             self.naive_session = self.get_naive_session()
 
-        return self._internal_request(self.naive_session, url, method, **kwargs)
+        return self._internal_request(self.naive_session, url, method, ignore401=False **kwargs)
```

Comment:
I see not comma behind the param:

Like this:
`return self._internal_request(self.naive_session, url, method, ignore401=False, **kwargs)`